examples/time/time.bal

```diff
@@ -2,7 +2,7 @@ import ballerina/io;
 import ballerina/time;
 
 public function main() returns error? {
-    // Get the current instant of the system clock(seconds from the epoch of
+    // Gets the current instant of the system clock(seconds from the epoch of
```

Comment:
```suggestion
    // Gets the current instant of the system clock (seconds from the epoch of
```

Reply:
Fixed in here https://github.com/ballerina-platform/ballerina-distribution/pull/1674/commits/7474f8ba8d119904de969e213366674d1dec8383